### PR TITLE
NPM package to fetch sessions details

### PR DESCRIPTION
**Pull request by Shivanshu-lambdatest**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@lambdatest/node-fetch-sessions",
+  "version": "1.0.0",
+  "description": "",
```
Comment (akshatlambdatest) [inline package.json]: Kindly add the description of the package.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const ltClient = require("@lambdatest/node-rest-client");
```
Comment (akshatlambdatest) [inline index.js:1]: Let's rename the file to `LTresults.js` as index.js will collide with other packages.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/LambdaTest/node-fetch-session.git"
+  },
+  "author": "LambdaTest <keys@lambdatest.com>",
+  "license": "ISC",
```
Comment (akshatlambdatest) [inline package.json]: We have added Apache 2.0 license, so lets update it

**Review by akshatlambdatest — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (Shivanshu-lambdatest) [reply index.js:1]: @akshatlambdatest index.js is specific to this package. Each package has it own index.js so it won't colide.
Also index.js conventionally gives sense of entry point that would be missing if we rename it to LTresult.js

File: package.json (hunk 1 above)
Comment (Shivanshu-lambdatest) [reply package.json]: Done

File: index.js (hunk 2 above)
Comment (Shivanshu-lambdatest) [reply index.js:1]: Defered

File: package.json (hunk 3 above)
Comment (Shivanshu-lambdatest) [reply package.json]: Done

**Review by akshatlambdatest — APPROVED** [review]

**Pull request merged**